### PR TITLE
Redirection links added at multiple place, Correction in the styling …

### PR DESCRIPTION
**Pull request by ydvirus**
Creating Pull request with feature/design because worked on latest feature/design and feature/design was not merged to develop. Now once Murtaza will create PR with develop with his branch, commit will also go into develop that time.

### CODE REVIEW

File: frontend/src/components/Home/Home.tsx [hunk 1]
```diff
@@ -48,24 +114,36 @@ export default function Home() {
...
             </Tab>
-            <Tab eventKey="womens" title="Women's Collection">
-              2
+            <Tab eventKey="womens" title={<><span>WOMEN'S</span><br/><span className='title-part-2'>Collection</span></>}>
+            <h1>TRENDING NOW</h1>
+              <Row>
+                {products.map((product, index) => {
+                  return (
+                    <Col md={3} key={index}>
+                      <ProductCard {...product} />
```
Comment (murtazamehmudji) [inline frontend/src/components/Home/Home.tsx:139]: Pass the prop like data={product} and in ProductCard define the Prop data as IProduct imported from product.model.ts

File: frontend/src/components/ProductCard/ProductCard.tsx [hunk 2]
```diff
@@ -1,9 +1,24 @@
-import React from 'react'
-
+import React from 'react';
 import './productcard.scss';
+import {Link } from 'react-router-dom';
+
+export default function ProductCard(props: any) {
```
Comment (murtazamehmudji) [inline frontend/src/components/ProductCard/ProductCard.tsx:5]: Import IProduct from product.model.ts

export default function ProductCard(props: {data: IProduct}) {

**Review by murtazamehmudji — CHANGES_REQUESTED** [review]

File: frontend/src/components/Home/Home.tsx (hunk 1 above)
Comment (ydvirus) [reply frontend/src/components/Home/Home.tsx]: Done

File: frontend/src/components/ProductCard/ProductCard.tsx (hunk 2 above)
Comment (ydvirus) [reply frontend/src/components/ProductCard/ProductCard.tsx]: Done

**Comment by ydvirus** [comment]
can merge now

**Comment by ydvirus** [comment]
Can merge now

**Pull request merged**